.github/workflows/ci.yml

```diff
@@ -590,6 +590,33 @@ jobs:
             composer config allow-plugins.silverstripe/vendor-plugin true
             composer config allow-plugins.phpstan/extension-installer true
 
+            # If --prefer-lowest is set, then use the lowest possible version of silverstripe/installer
+            if [[ "${{ matrix.composer_args }}" =~ --prefer-lowest ]]; then
```

Comment:
Do we need to have a check for if installer was there in the first place? Or do we not reach this code path for repos that shouldn't have installer installed?

Reply:
Good point, have added check to see if installer is used